common/Makefile.am

```diff
@@ -206,6 +206,19 @@ endif # ENABLE_DARK
 
 endif #!ENABLE_TRANSPARENCY
 
+	pwd
+	@if [ "$(GNOME_VERSION)" != "3.18" ]; then \
+		cd $(srcdir)/gtk-3.0/$(GNOME_VERSION) && cp -RL compile-gresources.sh $(ithemedir)/gtk-3.0/compile-gresources.sh;\
+		cd ../..;\
+		cd $(srcdir)/gtk-3.0/$(GNOME_VERSION) && cp -RL compile-gresources.sh $(ithemedarkdir)/gtk-3.0/compile-gresources.sh;\
+		cd ../..;\
+		cd $(srcdir)/gtk-3.0/$(GNOME_VERSION) && cp -RL compile-gresources.sh $(ithemedarkerdir)/gtk-3.0/compile-gresources.sh;\
+		cd ../..;\
+		cd $(ithemedir)/gtk-3.0/ && ./compile-gresources.sh LIGHT;\
+		cd $(ithemedarkdir)/gtk-3.0/ && ./compile-gresources.sh DARK;\
+		cd $(ithemedarkerdir)/gtk-3.0/ && ./compile-gresources.sh DARKER;\
+	fi
+
```

Comment:
The four instances of `GNOME_VERSION` here should be changed to `GTK_VERSION` after #117.

I'm also wondering what's the purpose of using `@if` here? The output isn't suppressed anywhere else in the script, and unless there's specific reason for this, it would make sense to stick with that practice.

Reply:
instances changed.  if is required to stop compilation for GTK+3.18 - compile-gresources is buggy for this version.